addon/test/test-metrics.js

```diff
@@ -28,12 +28,18 @@ const mocks = {
     Events: ['on', 'off', 'emit'],
     TelemetryController: ['submitExternalPing'],
     AddonManager: ['getAddonByID'],
+    request: ['post'],
```

Comment:
`sendGAEvent` is such a simple function that the linter is probably all the testing it needs at the moment, but for completeness we probably ought to write a test case in here for it. Its not a blocker IMO though.